src/components/user.rs

```diff
@@ -390,10 +390,10 @@ impl Wallet {
         Ok(())
     }
 
-    /// Add note data for those notes that are decryptable with one of this wallet's
-    /// incoming viewing keys to the wallet, and return a data structure that describes
-    /// the actions that are involved with this wallet, either spending notes belonging
-    /// to this wallet or creating new notes owned by this wallet.
+    /// Add note data for those notes that are decryptable with one of this user's
+    /// incoming viewing keys to the user, and return a data structure that describes
+    /// the actions that are involved with this user, either spending notes belonging
+    /// to this user or creating new notes owned by this user.
```

Comment:
"incoming viewing keys ~~to the user~~,"
Consider rewriting this entire comment to make it simpler and consistent and less weird